public/app/features/alerting/unified/mocks/server/entities/alertmanagers.ts

```diff
@@ -24,6 +29,11 @@ export const setupAlertmanagerConfigMapDefaultState = () => {
  */
 export const setAlertmanagerConfig = (alertmanagerName: string, config: AlertManagerCortexConfig) => {
   ALERTMANAGER_CONFIG_MAP.set(alertmanagerName, config);
+
+  if (alertmanagerName === GRAFANA_RULES_SOURCE_NAME) {
+    const routingTree = getUserDefinedRoutingTree(config);
+    setRoutingTree(ROOT_ROUTE_NAME, routingTree);
+  }
```

Comment:
(We should soon move to msw-data or similar, as maintaining the mock entities is a bit fiddly